k8s-watcher/server_routes.go

```diff
@@ -8,33 +8,60 @@ import (
 
 // routers registers the handlers for all http endpoints the server supports.
 func (s *Server) registerRoutes() {
-	s.router.HandlerFunc("GET", "/servers", s.handleIndex)
-	s.router.HandlerFunc("GET", "/servers/:serverID", s.handleServerID)
-	s.router.HandlerFunc("GET", "/users/:userID/servers", s.handleUserID)
-	s.router.HandlerFunc("GET", "/users/:userID/servers/:serverID", s.handleUserIDServerID)
+	s.router.HandlerFunc("GET", "/servers", s.jsGetAll)
+	s.router.HandlerFunc("GET", "/servers/:serverID", s.jsGetOne)
+	s.router.HandlerFunc("GET", "/users/:userID/servers", s.jsUserID)
+	s.router.HandlerFunc("GET", "/users/:userID/servers/:serverID", s.jsUserIDServerID)
+	s.router.HandlerFunc("GET", "/sessions", s.asGetAll)
+	s.router.HandlerFunc("GET", "/sessions/:serverID", s.asGetOne)
+	s.router.HandlerFunc("GET", "/users/:userID/sessions", s.asUserID)
+	s.router.HandlerFunc("GET", "/users/:userID/sessions/:serverID", s.asUserIDServerID)
```

Comment:
Note: the routing scheme may be confusing :sweat_smile: .

Reply:
I agree. But I was thinking that we will purge the old notebooks endpoints soon and after that it will be better. I can add a comment to explain the difference between server/sessions.

Reply:
"Soon" will not be soon. Renku 1.0 will have to be supported for a long time still.